classes/Router/Router.php

```diff
@@ -119,6 +119,6 @@ public function handle(Request $request)
 
         $method = $route['method'];
 
-        return (new $route['controller']($this->upgradeContainer))->$method($request);
```

Comment:
you prefer to pass $request to the constructor but if you check you already pass it to $method. I think we have to choose one, we don't need to pass 2 time the same variable to the Class.

Reply:
There is a topic to discuss here indeed. It is redundant to have it both in the constructor and the method, so what do we prefer:

I think it is nice to have the simplest possible method `$this->redirectTo($route)`, but this requires to have the request defined at the class level. On the other hand, I don't know if there is a potential gain to have the $request at the method level (I doubt it but could a same controller be called several times in different Request instances?).


I would be in favor of moving it in the constructor, but that's something we should all agree on.